internal/dinosql/gen.go

```diff
@@ -1145,6 +1145,12 @@ const (
 	{{- end}}
 )
 
+var every{{.Name}} = map[{{.Name}}]bool{
```

Comment:
I don't want to add this map by default. Can you put it behind a feature flag? `emit_enum_mapping` or something like that. Also, I think this should be public, as by default I don't encourage people to add code to the generated package.

Reply:
Done